specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/examples/CosmosDBSqlRoleAssignmentCreateUpdate.json

```diff
@@ -0,0 +1,28 @@
+{
+  "parameters": {
```

Comment:
Just to understand, what is the significance of these parameters?

Reply:
The param values were taken from the PDF examples. I've made them more generic for simplicity. Does this work better?